android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/AddonManager.kt

```diff
@@ -134,8 +134,8 @@ class AddonManager(
             id = addon.id,
             url = addon.downloadUrl,
             onSuccess = { ext ->
-                val installedAddon = addon.copy(installedState = ext.toInstalledState())
-                addonUpdater.registerForFutureUpdates(installedAddon.id)
+                val installedAddon = Addon.newFromWebExtension(ext, ext.toInstalledState())
```

Comment:
We don't need to pass `ext.toInstalledState()` anymore, right?

Reply:
We haven't land that [change](https://github.com/mozilla-mobile/firefox-android/blob/a25185f9967e544213f2333b1a1d45591940e7bd/android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/Addon.kt#L293) yet 😅

Reply:
hahaha